### DISCUSSION TITLE
Fail to make URL rewrite work

### DISCUSSION THREAD

**Discussion by nltesown** [vercel/community#5847] [Help] [answered]
### Summary

I have a basic SvelteKit project, with no custom configuration except a `/vercel.json` file that I created with this content:

```json
{
  "rewrites": [
    {
      "source": "/expo/:path*",
      "destination": "/:path*"
    }
  ]
}
```

This project is linked to a custom domain. I expect the rewrite rule to route requests at www.mydomain.com/expo to the app's root.
It doesn't work: what happens instead it that my app's entry page is at www.mydomain.com (and www.mydomain.com/expo responds with code 404) – in other words, the rule has no effect.

I don't see anything in the Vercel project's dashboard where I could check if the rewrite is taken into account.
Help much appreciated!

### Steps to Reproduce

(Not applicable)

**Comment by swarnava** [comment, marked as answer]
Sveltekit doesn't support rewrites using vercel.json: https://vercel.com/docs/edge-network/rewrites#using-rewrites-with-supported-frameworks

You may be able to make use of `@vercel/routing-utils`:https://vercel.com/docs/build-output-api/v3/features#high-level-routing

Alternatively, you can also leverage native `reroute` hook of Sveltekit: https://kit.svelte.dev/docs/hooks#universal-hooks-reroute

**Reply by nltesown** [reply]
Thanks for your prompt answer and for providing possible solutions. Very useful.
